pkg/puppet/puppet.go

```diff
@@ -170,6 +170,9 @@ func kubernetesClusterConfigPerRole(conf *clusterv1alpha1.ClusterKubernetes, rol
 		if conf.ClusterAutoscaler.Version != "" {
 			hieraData.variables = append(hieraData.variables, fmt.Sprintf(`kubernetes_addons::cluster_autoscaler::version: "%s"`, conf.ClusterAutoscaler.Version))
 		}
+
+		hieraData.variables = append(hieraData.variables, fmt.Sprintf(`kubernetes_addons::cluster_autoscaler::scale_down_utilization_threshold: %v`, *conf.ClusterAutoscaler.ScaleDownThreshold))
```

Comment:
I think a check is needed here in case ScaleDownThreshold is nil?

Reply:
Shouldn't need a check since it's set in defaults but couldn't hurt

Reply:
Ah yeah, good point